Questions about risc-v vector support

Hi, I notice that openblas support riscv vector support, if I don't misunderstand, the vector support are only available for C910V,
but my processor is C906 which supports vector instructions 0.71, so can I just replace `C910V` in the file with `C906` to make
it support my processor.

Once, I try to cross compile for C910V, but get the error cross compiler doesn't support flag `lp64v`, it only supports `lp64` , `lp64d` , do I need to recompile my cross compiler to support `lp64v`, Thanks!

Thanks, I have tested the cross compiler in demos, it has some errors
```
../kernel/riscv64/amax_vector.c:34:19: error: unknown type name 'vfloat32m8_t'; did you mean 'float32xm8_t'?
 #define FLOAT_V_T vfloat32m8_t
                   ^~~~~~~~~~~~
../kernel/riscv64/amax_vector.c:34:19: note: in definition of macro 'FLOAT_V_T'
 #define FLOAT_V_T vfloat32m8_t
                   ^~~~~~~~~~~~
../kernel/riscv64/amax_vector.c:35:22: error: unknown type name 'vfloat32m1_t'; did you mean 'float32xm1_t'?
 #define FLOAT_V_T_M1 vfloat32m1_t
```
I guess it is because the compiler is too old, I will find a new version C910V compiler to try.

I have updated to the new C910V compiler, but when run tests, it gets the error:
```
/home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/bin/ld: (.text._gfortrani_set_fpu_trap_exceptions+0xa6): warning: feenableexcept is not implemented and will always fail
/home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/bin/ld: /home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/lib/../lib64vxthead/lp64v/libgfortran.a(fpu.o): in function `_gfortrani_get_fpu_trap_exceptions':
(.text._gfortrani_get_fpu_trap_exceptions+0x12): warning: fegetexcept is not implemented and will always fail
/home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/bin/ld: ../libopenblas_c910vp-r0.3.15.dev.a(memory.o): in function `alloc_mmap':
memory.c:(.text+0x78): undefined reference to `pthread_mutex_lock'
/home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/bin/ld: memory.c:(.text+0xa2): undefined reference to `pthread_mutex_unlock'
/home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/bin/ld: ../libopenblas_c910vp-r0.3.15.dev.a(memory.o): in function `.L60':
memory.c:(.text+0x33a): undefined reference to `pthread_mutex_lock'
/home/hermanhe/software/riscv64-linux-x86_64/bin/../lib/gcc/riscv64-unknown-linux-gnu/8.4.0/../../../../riscv64-unknown-linux-gnu/bin/ld: ../libopenblas_c910vp-r0.3.15.dev.a(memory.o): in function `.L0 ':
memory.c:(.text+0x362): undefined reference to `pthread_mutex_unlock'
```
any advice, thanks!

--- a/test/Makefile
+++ b/test/Makefile
@@ -259,10 +259,6 @@ endif
 
 FLDFLAGS = $(FFLAGS:-fPIC=) $(LDFLAGS)
 
-ifeq ($(CORE), C910V)
-EXTRALIB =
-CEXTRALIB =
-endif
 
 ifeq ($(USE_OPENMP), 1)
 ifeq ($(F_COMPILER), GFORTRAN)

